docs/src/lib/functions.md

```diff
@@ -0,0 +1,86 @@
+```@meta
+CurrentModule = DataFrames
+```
+
+# Functions
+
+```@index
+Pages = ["functions.md"]
+```
+
+## Joins
+
+```@docs
+join
+```
+
+## Reshaping
+
+```@docs
+melt
+stack
+unstack
+stackdf
+meltdf
+```
+
+## Data Manipulation
+
+```@docs
+aggregate
+by
+colwise
+groupby
+```
+
+## DataFrame Manipulation
```

Comment:
`combine` should be in the previous section. I'm not sure how to describe this set of functions, but the current title isn't descriptive enough IMHO. I find it hard to find meaningful groupings for all functions.